docs/UserGuide.md

```diff
@@ -14,23 +15,25 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
 1. Ensure you have Java `11` or above installed in your Computer.
 
-1. Download the latest `addressbook.jar` from [here](https://github.com/se-edu/addressbook-level3/releases).
+1. Download the latest `mcscheduler.jar` from [here](https://github.com/AY2021S1-CS2103-F10-4/tp/releases).
 
-1. Copy the file to the folder you want to use as the _home folder_ for your AddressBook.
+1. Copy the file to the folder you want to use as the _home folder_ for McScheduler.
 
 1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
    ![Ui](images/Ui.png)
 
 1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
    Some example commands you can try:
 
-   * **`list`** : Lists all contacts.
+   * **`worker list`** : Lists all workers in the McScheduler.
+
+   * **`worker add`**`n/John hp/98765432 a/21 Lower Kent Ridge Rd, Singapore 119077 r/Cashier p/7` : Adds a cashier named John whose pay is $7/hr. His phone number is 98765432 and he lives at 21 Lower Kent Ridge Rd, Singapore 119077.
 
-   * **`add`**`n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the Address Book.
+   * **`worker delete`**`3` : Deletes the 3rd worker shown in the worker list.
 
-   * **`delete`**`3` : Deletes the 3rd contact shown in the current list.
+   * **`shift add`**`d/Wed t/AM r/Cashier-2 r/Cleaner-3` : Adds a shift on Wednesday morning, which requires 2 workers to fill the cashier role and 3 workers to fill the cleaner role.
 
-   * **`clear`** : Deletes all contacts.
+   * **`assign`**`s/3 w/2 r/Cashier`: Assign the 2nd worker on the list to the 3rd shift on the list as a Cashier.
```

Comment:
```suggestion
   * **`assign`**`s/3 w/2 r/Cashier` : Assign the 2nd worker on the list to the 3rd shift on the list as a Cashier.
```
Inconsistent spacing around `:` compared to the rest of the bullet points

Reply:
Thanks